packages/utils/src/common/sharding/routing_info.ts

```diff
@@ -0,0 +1,88 @@
+import type {
+  AutoSharding,
+  ContentTopic,
+  IRoutingInfo,
+  NetworkConfig,
+  PubsubTopic,
+  ShardId,
+  StaticSharding
+} from "@waku/interfaces";
+
+import {
+  contentTopicToShardIndex,
+  ensureValidContentTopic,
+  formatPubsubTopic,
+  pubsubTopicToSingleShardInfo
+} from "./index.js";
+
+export class RoutingInfo implements IRoutingInfo {
```

Comment:
I would keep it in `waku/core` - it is kinda important abstraction for all parts
also, I was thinking of making `ShardReader` doing something similar so maybe later these concepts can be combined

Reply:
I put it where all the shard utils where, I'll see if I can move it

Reply:
we can iterate on it, I think shard utils shouldn't exist when we mature this part of code